server/app.js

```diff
@@ -55,6 +55,34 @@ app.use(bodyParser.json({ limit: '25mb' }));
 app.use(cookieParser());
 app.use(forceSSL);
 
+import cors from 'cors';
+
+// CORS Configuration
+const corsOptions = {
+    origin: (origin, callback) => {
+        const allowedOrigins = [
+            'https://homebrewery.naturalcrit.com',
+            'http://localhost:8000',
```

Comment:
Is it possible for someone to send rogue commands from their local machine and maliciously rename authors without their permission?

Reply:
I should be able to change the allowed list based on the homebrewery domain as well.

Reply:
patched, localhost will be added only if on local